modules/core/constants.js

```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line
-module.exports = require('./dist/es5/constants');
+console.warn('"import `luma.gl/constants`"" has been replaced with "import `@luma.gl/constants`"');
+require('@luma.gl/connstants');
```

Comment:
- Typo
- Still need to export it?

Reply:
Shouldn't this be added to the dependencies of the core package?

Reply:
Good point, done.